avocado/utils/iso9660.py

```diff
@@ -82,8 +82,9 @@ def can_mount():
 
     :rtype: bool
     """
-    if os.getuid() != 0:
-        logging.debug('Can not use mount: current user is not "root"')
+    if not process.sudo_available():
+        logging.debug('Can not use mount: current user is not "root" and'
```

Comment:
If you choose to change this to `has_superuser_privileges` (as commented on previous commit), the message could be changed accordingly.